ci-operator/config/maistra/istio/maistra-istio-maistra-2.4.yaml

```diff
@@ -0,0 +1,55 @@
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.4"
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      memory: 16Gi
+    requests:
+      cpu: "4"
+      memory: 4Gi
+tests:
+- as: istio-unit-2-4
+  commands: |
+    make -e T=-v build racetest binaries-test \
+    BUILD_WITH_CONTAINER="0" \
+    GOBIN="/gobin" \
+    GOCACHE="/tmp/cache" \
+    GOMODCACHE="/tmp/cache" \
+    GOFLAGS="-mod=vendor" \
+    XDG_CACHE_HOME="/tmp/cache" \
+    ISTIO_ENVOY_BASE_URL="https://storage.googleapis.com/maistra-prow-testing/proxy"
+  container:
+    from: src
+  timeout: 4h0m0s
+- as: istio-lint-2-4
+  commands: |
+    make lint \
+    BUILD_WITH_CONTAINER="0" \
+    GOBIN="/gobin" \
+    GOCACHE="/tmp/cache" \
+    GOMODCACHE="/tmp/cache" \
+    GOFLAGS="-mod=vendor" \
+    XDG_CACHE_HOME="/tmp/cache" \
+    ISTIO_ENVOY_BASE_URL="https://storage.googleapis.com/maistra-prow-testing/proxy"
+  container:
+    from: src
+- as: istio-gencheck-2-4
+  commands: |
+    make gen-check \
+    BUILD_WITH_CONTAINER="0" \
+    GOBIN="/gobin" \
+    GOCACHE="/tmp/cache" \
+    GOMODCACHE="/tmp/cache" \
+    GOFLAGS="-mod=vendor" \
```

Comment:
and this

Reply:
We may not need them when we have   use_build_cache: true in the config yaml.
In the past, I had job failures without those Go env variables before I set the   use_build_cache: true.
We need to test if we still have some failures without Go env variables running in an openshift pod.